### DISCUSSION TITLE
`Class "Appwrite\Migration\Version\V13" not found` when using the Appwrite CLI to create a function

### DISCUSSION THREAD

**Discussion by itsarvinddev** [appwrite/appwrite#5087] [Q&A]
### 👟 Reproduction steps

Open terminal
Run the command `appwrite init function`
Observe the error message

<img width="526" alt="Screenshot 2023-02-05 at 3 54 26 PM" src="https://user-images.githubusercontent.com/59143934/216814020-8ad4288c-baec-43a1-928a-38985f07ad2d.png">

 **OR**
 
![Screenshot 2023-01-28 at 5 13 06 PM](https://user-images.githubusercontent.com/59143934/216814046-f8ce094d-5e37-47b0-96f9-e29e892f1414.png)

### 👍 Expected behavior

A new function should be created successfully without any errors.

### 👎 Actual Behavior

The server crashes and throws the error message: `Error Error: Server Error` in terminal.
After this server not working and appwrite console show error: `{"message":"Error: Server Error","code":500,"version":"1.2.0"}`

<img width="552" alt="Screenshot 2023-02-05 at 4 14 46 PM" src="https://user-images.githubusercontent.com/59143934/216814514-1d39dafb-2164-4510-aff2-479dabaccda2.png">

### 🎲 Appwrite version

Version 1.2.x

### 💻 Operating system

Linux

### 🧱 Your Environment

I use digitalocean server.

Discord server chat regarding to this issue: https://discord.com/channels/564160730845151244/812714347298160690/1068849702885396561

`docker-compose logs appwrite` Logs:
<img width="635" alt="Screenshot 2023-02-05 at 4 24 27 PM" src="https://user-images.githubusercontent.com/59143934/216814906-82821903-8e5e-4b53-beb6-a71c816fe88e.png">

Please let me know if there is a solution or workaround for this issue. I would be happy to provide any additional information or logs that could be helpful in resolving this problem.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
What version of the Appwrite CLI do you have? You should be able to run `appwrite -v` to check the version. With Appwrite version 1.2.0, version 1.1.1 of the Appwrite CLI should work.

**Reply by itsarvinddev** [reply]
I have Appwrite CLI version 0.15.0

**Reply by stnguyen90** [reply]
That version is quite old. For Appwrite version 1.2.0, you should be using Appwrite CLI version 1.1.1. Please try to upgrade.
